schedule/publiccloud/publiccloud_cli.yml

```diff
@@ -7,5 +7,5 @@ schedule:
   - publiccloud/azure_cli
   - publiccloud/aws_cli
   # We currently don't package gcloud binary
-  # - publiccloud/google_cli
+  - publiccloud/google_cli
```

Comment:
can you elaborate why you think we should change this ? ( keeping in mind all discussions around gcloud in previous PR )

Reply:
I think that we should at least keep checking if nothing breaks from our side.
Even if the package is not our it can interfere with something else.

Reply:
ok